src/main/resources/init_db.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE IF NOT EXISTS books (
+    id BIGINT NOT NULL AUTO_INCREMENT,
```

Comment:
primary key is not null by default
```suggestion
    id BIGINT AUTO_INCREMENT,
```